src/app/public/modules/viewkeeper/viewkeeper-host-options.ts

```diff
@@ -7,4 +7,16 @@ import {
 } from './viewkeeper-options';
 
 @Injectable()
-export class SkyViewkeeperHostOptions implements SkyViewkeeperOptions { }
+export class SkyViewkeeperHostOptions implements SkyViewkeeperOptions {
+  public setWidth?: boolean;
+
+  public el?: HTMLElement;
+
+  public boundaryEl?: HTMLElement;
+
+  public verticalOffset?: number;
+
+  public verticalOffsetEl?: HTMLElement;
+
+  public viewportMarginTop?: number;
```

Comment:
☝️Code-style > public properties should be in alphabetical order. Other than that, LGTM.